server.js

```diff
@@ -23,43 +24,38 @@ const port = process.env.PORT || 3000
 
 app.prepare().then(() => {
   createServer((req, res) => {
-    const parsedUrl = parse(req.url, true)
-    const { pathname, query } = parsedUrl
+    const parsedUrl = parseURL(req.url)
+    const { pathname, queryStr } = parsedUrl
```

Comment:
There's no `queryStr` when parsing a URL. The querystring is called "query" just the same. The difference is that it's an object if `url.parse`'s second argument is true.

Reply:
Thanks! Yes, this is part of the problem (my bad, fixed in b85eb64). But the actual issue is a bug in Next, it seems. See https://github.com/zeit/next.js/issues/7303#issuecomment-511053550

Addressing... ⏳